### DISCUSSION TITLE
Missing variable "groopsDataDir"

### DISCUSSION THREAD

**Discussion by RoyanLiSu** [groops-devs/groops#148] [Q&A] [answered]
Dear developers,
I installed the software on windows. when i wanted to run a config file for example scenarioGraceGravityfieldRecovery, i forced with this error . how can i fixed it ?

Thank you in advance for your time and help.
With Kind Regards

![Untitled](https://user-images.githubusercontent.com/110038504/215319634-aa5dd7f5-ef3a-44e8-8418-8f3308bfbc66.jpg)

**Comment by tmayerguerr** [comment, marked as answer]
Dear RoyanLiSu,

You should add a variable "groopsDataDir" (fileName) in the global section with the path of your local
GROOPS data directory (available at https://ftp.tugraz.at/outgoing/ITSG/groops/)
(e.g. groopsDataDir=C:\groops\data).

regards
Torsten

**Comment by RoyanLiSu** [comment]
Dear Torsten,
I am very appreciative of your response. I did it and it was successful for simulate the orbit and the star camera. but I still received an error for simulate accelerometer.
is the error about config in first line of errors make a problem for the results? 
Thank you for your time and help.

![error2](https://user-images.githubusercontent.com/110038504/215321554-71ecde54-bde4-4c54-addc-276efbfb3a8e.png)

**Comment by tmayerguerr** [comment]
Please read the error message: You select the FES2014b ocean tide model but select the admittance file for the model EOT11a.

**Reply by RoyanLiSu** [reply]
yes, you are right. I am very grateful for your help.
best regards
RoyanLiSu
